layouts/_default/baseof.html

```diff
@@ -2,6 +2,7 @@
 <html lang="en">
   <head>
     {{- partial "head.html" . -}}
+    {{- partial "redirect.html" . -}}
```

Comment:
I'll be honest that I think I preferred it when redirect was its own layout instead of directly being inserted into `_default`, because this feels like a gun pointed at our feet.

With it being its own, at least you'd have to purposefully select the layout with `type` in the front matter. Sorry to make you go back in work on this, but I think this is an unsafe solution.

Reply:
> It might also be worth to not override baseof, but instead add to the headers and body directly, so if it gets stuck on the page, at least it's not a blank white page with some html

Based on the comment above from earlier, I was under the impression that we do not want to override the base template. I can change it back but I would like to confirm that we _do_ want this inside a separate template with a similar layout.

Reply:
I worded that poorly; I was implying that we should *duplicate* `baseof`, but re-reading myself I did not make it clear. I think in 99% of the cases that overriding the base html template is probably bad, except in cases like:

1. It fundamentally changes how the page works (in your case, making it so that a single front-matter piece makes the page change from a static page affair into a redirect)
2. It requires the layout/style/etc to be so different that it would be a dirty hack to make work (in [my](https://github.com/The-Balance-FFXIV/glam/pull/299) case with the dynamically generated editor bits)